RawSpeed/RawDecoder.cpp

```diff
@@ -552,6 +552,20 @@ void RawDecoder::setMetaData(CameraMetaData *meta, string make, string model, st
     }
   }
 
+  // Allow overriding the whitebalance. Values are R,G,B multipliers
+  // A hint could be:
+  // <Hint name="override_whitebalance" value="10,20,30"/>
+  if (cam->hints.find(string("override_whitebalance")) != cam->hints.end()) {
```

Comment:
I've found two cases where this makes sense. The Nikon D1 applies the WB to the raw data, so the WB is fixed even though the camera reports the values that were applied. For that case we could probably reverse the WB and then report the actual WB. Since all files from this camera are like this I didn't bother doing that.

The Mamiya ZD doesn't seem to have WB metadata at all and for those cases dcraw sets something calculated from the color matrix which is what I've set here. This will also be needed for the cameras in PR #44 that have absolutely no metadata and thus would need something sensible to be set. If you instead decide to include the camera matrix in cameras.xml (to fix sNEF) we could do the same calculation dcraw does and not need to hardcode the WB here.